### ISSUE TITLE
Can you provide details on setting options about lite-mono-tiny,lite-mono-small and the ablation expriment?

### ISSUE THREAD

**Issue by lnSong** [noahzn/Lite-Mono#17]
Can you provide details on setting options about lite-mono-tiny,lite-mono-small and the ablation expriment?
the option such as learning rate,epoch and batch size...
Thank you!

**Comment by lnSong** [comment]
Are the learning rate and epoch settings in the options for lite mono tiny, lite mono small, and lie mono the same?

**Comment by noahzn** [comment]
Yes, they are the same. You will find that all the models using the pre-trained weights converge quickly, and please check all the epochs to get the best model.
